src/main/java/com/marklogic/appdeployer/command/schemas/LoadSchemasCommand.java

```diff
@@ -0,0 +1,51 @@
+package com.marklogic.appdeployer.command.schemas;
+
+import java.io.File;
+
+import com.marklogic.appdeployer.AppConfig;
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.schemasloader.SchemasLoader;
+import com.marklogic.client.schemasloader.impl.DefaultSchemasFinder;
+import com.marklogic.client.schemasloader.impl.DefaultSchemasLoader;
+
+public class LoadSchemasCommand extends AbstractCommand  {
+
+	private SchemasLoader schemasLoader;
+	
+	public LoadSchemasCommand() {
+        setExecuteSortOrder(SortOrderConstants.LOAD_SCHEMAS);
+	}
+	
+	@Override
+	public void execute(CommandContext context) {
+		loadSchemasIntoSchemasDatabase(context);
+	}
+
+	protected void loadSchemasIntoSchemasDatabase(CommandContext context) {
+		if (schemasLoader == null) {
+			initializeDefaultSchemasLoader(context);
+		}
+		
+		
+		AppConfig config = context.getAppConfig();
+        DatabaseClient client = config.newSchemasDatabaseClient();
+        try {
+        	File baseDir = config.getConfigDir().getBaseDir();
```

Comment:
One other idea - I think AppConfig should have a "getSchemasPath" method with a "private String schemasPath" property, similar to private List<String> modulePaths. 

This allows ml-gradle to then support something like an mlSchemasPath property in gradle.properties, and if it's set, it'll modify LoadSchemasCommand.schemasPath. That gives the Gradle user full control over the path to schemas. 

I think regardless of where the default path is for schemas, we should have this property so that it's easy to customize.